libs/jump_portrait/src/jump_portrait/fetch.py

```diff
@@ -119,6 +120,52 @@ def get_jump_image(
     return result
 
 
+def get_jump_image_iter(metadata: pl.DataFrame, channel: List[str],
+                        site: List[str], correction:str='Orig',
+                        print_progress: bool=True,
+                        ) -> (pl.DataFrame, List[tuple]):
+    '''
+       Load jump image associated to metadata in a threaded fashion.
+        ----------
+    Parameters:
+        metadata(pl.DataFrame): must have the shape ("Metadata_Source",
+    "Metadata_Batch", "Metadata_Plate", "Metadata_Well")
+        channel(List[str]): list of channel desired
+            Must be in ['DNA', 'ER', 'AGP', 'Mito', 'RNA']
+        site(List[str]): list of site desired
+            For compound, must be in ['1' - '6']
+            For ORF, CRISPR, must be in ['1' - '9']
+        correction(str): Must be 'Illum' or 'Orig'
```

Comment:
The docstrings should be easy to read for humans, thus syntax like List[str] is not ideal, replace them with 'list of strings'